Automattic/jetpack
# Jetpack: Change responses export filename

Fixes #28398 

## Proposed changes:
This PR changes the file/spreadsheet name when exporting form responses.

The exported file or spreadsheet name will now follow this template:

[ Site name ] - Jetpack Form Responses - [ source (if any) ] - [ current date ]

`current date` in the ISO 8601 format, YYYY-MM-DD HH:MM.

`source` is the post title where the form is published, if the export is done with this filter.


### Other information:


## Jetpack product discussion
pcsBup-K4-p2


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Create a post with a form, visit it and submit a few responses. On wp-admin, go to Feedback -> Form Responses.

Try exporting to CSV. See the filename complies to the described template. Apply a source filter and export again, the filename should include the post tile from the filter.
